src/Payum/GatewayFactory.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+namespace AdeoWeb\SyliusEveryPayPlugin\Payum;
+
+use Payum\Core\Bridge\Spl\ArrayObject;
+use Payum\Core\GatewayFactory as BaseFactory;
+
+final class GatewayFactory extends BaseFactory
+{
+    protected function populateConfig(ArrayObject $config): void
+    {
+        $config->defaults([
+            'payum.factory_name' => 'everypay',
+            'payum.factory_title' => 'EveryPay',
+        ]);
+
+        $config['payum.api'] = static function (ArrayObject $config): EveryPayConfiguration {
+            return new EveryPayConfiguration(
+                $config['api_url'],
```

Comment:
čia gal null coalescing operatorių panaudot visur, just in case? arba irgi symfonio denormalizerį čia panaudot. paduoti ArrayObject kaip array, o denormalizeris, kadangi array raktai atitinka konstruktoriaus argumentų pavadinimus, per konstruktorių viską sukiš kur reikia. o jeigu kas nors nepaeis, čia exceptionas bus

Reply:
Uždėjau coalescingus.

Reply:
Nors dabar nuimsiu, nes prieš pat šitą inicijavimą dabar pravaliduojama, kad visi tie laukai būtų netušti.